addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js

```diff
@@ -97,21 +97,24 @@ class EnvironmentScService extends Service {
       });
 
     if (this.isAppStreamEnabled()) {
-      envs = await this.filterAppStreamProjectEnvs(requestContext, envs);
+      envs = await this.markAppStreamConfig(requestContext, envs);
     }
 
     return this.augmentWithConnectionInfo(requestContext, envs);
   }
 
-  async filterAppStreamProjectEnvs(requestContext, envs) {
+  async markAppStreamConfig(requestContext, envs) {
```

Comment:
Nitpick: Lets name this to `markAppStreamConfigured`.

`markAppStreamConfig` makes me think of `markAppStreamConfiguration`, which doesn't make sense.